sklearn/cluster/_mean_shift.py

```diff
@@ -379,6 +380,16 @@ class MeanShift(ClusterMixin, BaseEstimator):
     MeanShift(bandwidth=2)
     """
 
+    _parameter_constraints = {
+        "bandwidth": [Real, None],
```

Comment:
According to the error raised on lines 433-435, bandwidth should be > 0.
You can now remove the validation of bandwidth on lines 433-435

```suggestion
        "bandwidth": [Interval(Real, 0, None, closed="neither"), None],
```